src/frontend/src/components/question/QuestionEdit.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div class="question-form">
+    <div class="inner">
+      <v-card max-width="700" color="#CFE1E8" class="inner-card">
+        <v-text-field
+          class="input-box"
+          label="질문 제목"
+          counter="30"
+          v-model="title"
+          autofocus
+          required
+        ></v-text-field>
+        <v-textarea
+          class="input-box"
+          label="질문 내용"
+          v-model="content"
+          counter
+          no-resize
+          clearable
+          rows="2"
+          row-height="1"
+          required
+        ></v-textarea>
+        <div class="control-box">
+          <hashtag-box @hashtags="receiveHashtags"></hashtag-box>
+          <form-button>
+            <button class="button" @click="onUpdateQuestion">
+              <span>수정하기</span>
+            </button>
+          </form-button>
+        </div>
+      </v-card>
+    </div>
+  </div>
+</template>
+
+<script>
+import HashtagBox from "./HashtagBox";
+import FormButton from "./FormButton";
+import { mapGetters } from "vuex";
+
+export default {
+  data() {
+    return {
+      questionId: this.$route.params.id,
+      title: "",
+      content: "",
+      hashtags: []
+    };
+  },
+  computed: {
+    ...mapGetters(["fetchedLoginUser", "fetchedQuestion"])
+  },
+  methods: {
+    async onUpdateQuestion() {
+      await this.$store.dispatch("UPDATE_QUESTION", {
+        title: this.title,
+        content: this.content,
+        hashtags: this.hashtags,
+        questionId: this.questionId
+      });
+      await this.$router.push(`/questions/${this.questionId}`);
+    },
+    receiveHashtags(hashtags) {
+      this.hashtags = hashtags;
+    }
+  },
+  watch: {
+    fetchedLoginUser: function() {
+      this.title = this.fetchedQuestion.title;
+      this.content = this.fetchedQuestion.content;
+      this.hashtags = this.fetchedQuestion.hashtags;
+    }
```

Comment:
methods, watch 등에서 함수를 정의할 때는 function은 생략하고 아래와 같이 쓰는걸로 통일하는게 좋을 것 같아요!

```suggestion
    fetchedLoginUser() {
      this.title = this.fetchedQuestion.title;
      this.content = this.fetchedQuestion.content;
      this.hashtags = this.fetchedQuestion.hashtags;
    }
```

Reply:
넵 반영했습니다~